src/datumaro/plugins/data_formats/arrow/importer.py

```diff
@@ -3,63 +3,71 @@
 # SPDX-License-Identifier: MIT
 
 import os
-import re
 from typing import Dict, List, Optional
 
 import pyarrow as pa
 
+from datumaro.components.errors import DatasetImportError
 from datumaro.components.format_detection import FormatDetectionConfidence, FormatDetectionContext
 from datumaro.components.importer import Importer
 
 from .format import DatumaroArrow
 
+__all__ = ["ArrowImporter"]
+
+
+def _verify_datumaro_arrow_format(file: str) -> None:
```

Comment:
Why was this defined as a global method?

Reply:
Move it to the static member function of `ArrowImporter`.